component/otelcol/processor/filter/filter.go

```diff
@@ -27,9 +27,9 @@ func init() {
 type Arguments struct {
 	// ErrorMode determines how the processor reacts to errors that occur while processing a statement.
 	ErrorMode ottl.ErrorMode `river:"error_mode,attr,optional"`
-	Traces    traceConfig    `river:"traces,block,optional"`
-	Metrics   metricConfig   `river:"metrics,block,optional"`
-	Logs      logConfig      `river:"logs,block,optional"`
+	Traces    TraceConfig    `river:"traces,block,optional"`
+	Metrics   MetricConfig   `river:"metrics,block,optional"`
+	Logs      LogConfig      `river:"logs,block,optional"`
```

Comment:
Had to export these types so I could build the Arguments.

Reply:
👍 They should've been exported anyway :)